components/podIndicator/index.test.jsx

```diff
@@ -69,3 +78,30 @@ describe("closeHandler", () => {
     expect(setAnchorEl).toHaveBeenCalledWith(null);
   });
 });
+
+describe("copy pod uri", () => {
+  jest.setTimeout(3000);
```

Comment:
This shouldn't be necessary.

Reply:
agreed - you can take it out. I was trying a few things to see why tests might be passing locally but not here.